lib/moon_web/components/header.ex

```diff
@@ -0,0 +1,51 @@
+defmodule MoonWeb.Components.Header do
+  @moduledoc false
+
+  use MoonWeb, :stateless_component
+
+  prop click, :event
+  slot default
+
+  def render(assigns) do
+    ~F"""
+    <header>
+      <div class="relative z-10 py-4 lg:hidden flex flex-row gap-2 items-center justify-between text-bulma">
+        <button type="button" class="focus:outline-none" phx-click={@click}>
```

Comment:
Should it be :on-click otherwise target might be wrong?

Reply:
i'm using LiveView.JS events here, simple DOM manipulation. so target is parent liveView, and that's what's needed here bc "#left-menu-container" is outside Header component.